tests/pytests/test_multibyte_char_terms.py

```diff
@@ -0,0 +1,1443 @@
+
+    env.cmd('FT.CREATE', 'idx_no_stem', 'ON', 'HASH', 'LANGUAGE', 'RUSSIAN',
+            'SCHEMA', 't', 'TEXT', 'NOSTEM')
+
+    # Create sample data
+    # lowercase terms
+    conn.execute_command('HSET', 'doc:1L', 't', 'программирование') # programming
+    conn.execute_command('HSET', 'doc:2L', 't', 'программирования')
+    conn.execute_command('HSET', 'doc:3L', 't', 'программированию')
+    conn.execute_command('HSET', 'doc:4L', 't', 'программированием')
+    conn.execute_command('HSET', 'doc:5L', 't', 'программировании')
+    # mixed case terms
+    conn.execute_command('HSET', 'doc:1M', 't', 'ПРОГРАммирование')
+    conn.execute_command('HSET', 'doc:2M', 't', 'ПРОГРАммирования')
+    conn.execute_command('HSET', 'doc:3M', 't', 'ПРОГРАммированию')
+    conn.execute_command('HSET', 'doc:4M', 't', 'ПРОГРАммированием')
+    conn.execute_command('HSET', 'doc:5M', 't', 'ПРОГРАммировании')
+
+    # Search using the index with stemming
+    expected = [10, 'doc:1M', 'doc:4M', 'doc:5M', 'doc:3M', 'doc:2M',
+                    'doc:1L', 'doc:4L', 'doc:5L', 'doc:3L', 'doc:2L']
+    res = conn.execute_command(
+        'FT.SEARCH', 'idx_stem', 'программирование', 'NOCONTENT',
+        'LANGUAGE', 'RUSSIAN', 'SORTBY', 't')
+    env.assertEqual(res, expected)
+
+    res = conn.execute_command(
+        'FT.SEARCH', 'idx_stem', 'ПРОГРАммировании', 'NOCONTENT',
+        'LANGUAGE', 'RUSSIAN', 'SORTBY', 't')
+    env.assertEqual(res, expected)
+
+    # Search using the NO_STEM index
+    res = conn.execute_command(
+        'FT.SEARCH', 'idx_no_stem', 'программирование', 'NOCONTENT',
+        'SORTBY', 't')
+    env.assertEqual(res, [2, 'doc:1M', 'doc:1L'])
+
+    res = conn.execute_command(
+        'FT.SEARCH', 'idx_no_stem', 'ПРОГРАММИРОВАНИИ', 'NOCONTENT',
+        'SORTBY', 't')
+    env.assertEqual(res, [2, 'doc:5M', 'doc:5L'])
+
+def testGreekStemming(env):
+    '''Test stemming with multi-byte character words.'''
+    conn = getConnectionByEnv(env)
+    env.cmd('FT.CREATE', 'idx_stem', 'ON', 'HASH', 'LANGUAGE', 'GREEK',
+            'SCHEMA', 't', 'TEXT')
+
+    env.cmd('FT.CREATE', 'idx_no_stem', 'ON', 'HASH', 'LANGUAGE', 'GREEK',
+            'SCHEMA', 't', 'TEXT', 'NOSTEM')
+
+    # Create sample data
+    # lowercase terms
+    conn.execute_command('HSET', 'doc:1L', 't', 'αεροπλάνο') # airplane
+    conn.execute_command('HSET', 'doc:2L', 't', 'αεροπλάνα') # airplanes
+    conn.execute_command('HSET', 'doc:3L', 't', 'αεροπλάνου') # of airplane
+
+    # upper case terms
+    conn.execute_command('HSET', 'doc:1U', 't', 'ΑΕΡΟΠΛΆΝΟ')
+    conn.execute_command('HSET', 'doc:2U', 't', 'ΑΕΡΟΠΛΆΝΑ')
+    conn.execute_command('HSET', 'doc:3U', 't', 'ΑΕΡΟΠΛΆΝΟΥ')
+
+    # Search using the index with stemming
+    expected = [6, 'doc:2U', 'doc:1U', 'doc:3U', 'doc:2L', 'doc:1L', 'doc:3L']
+    res = conn.execute_command(
+        'FT.SEARCH', 'idx_stem', 'αεροπλάνα', 'NOCONTENT', 'SORTBY', 't',
+        'LANGUAGE', 'GREEK')
```

Comment:
Same here - in original PR there is no `LANGUAGE` specified here, but it has no effect here since it is the same as in the index definition, right?

Reply:
Yes. It's in the PR description